### ISSUE TITLE
Optative in τίθημι: Present System Middle-Passive

### ISSUE THREAD

**Issue by monzug** [alpheios-project/inflection-tables#343]
Some endings in the Optative for verbpdgm29 τίθημι: Present System Middle-Passive table are missing in Morpheus, except for the first person singular and the first person plural.
I did also check the Goodell grammar and there are the same endings as in Mastronarde tables. Morpheus or Mastronarde?

**Comment by monzug** [comment]
actually this optative table is identical to the one in Smith (page 125)

**Comment by balmas** [comment]
so @monzug it sounds like we should note the forms that are missed by Morpheus here?

**Comment by monzug** [comment]
Lexical query produced no results in Morpheus
τιθεῖο           2nd person singular optative
τιθεῖσθον    2nd person dual optative
τιθεῖσθε      2nd person plural optative
τιθεῖντο      3rd person plural optative

τιθεῖτο 3rd person singular optative goes to an other table as stemtype is ew_pr and not emi_pr

**Comment by vgorman1** [comment]
I don’t know what voice and tense you are looking for, but according to Diogenes, the only second person optatives of τίθημι that are attested are::
τιθείης: (pres opt act 2nd sg)
τιθείητε: (pres opt act 2nd pl)
τιθείητον: (pres opt act 2nd dual)
τιθεῖτε: (pres opt act 2nd pl)

Vanessa B. Gorman
Professor of Ancient History
Department of History
619 Oldfather Hall
University of Nebraska-Lincoln
Lincoln, NE 68588-0327
https://vgorman1.github.io/

**Comment by monzug** [comment]
yes, we have those endings in the active forms. the ones above are for the middle passive. Thank u!

**Comment by balmas** [comment]
So @monzug I wonder, should we have a policy that if they aren't attested in the TLG corpus per Diogenes, and morpheus doesn't have them, but the table does, that we just add a footnote to the forms in the table that says they may not be attested?

**Comment by monzug** [comment]
Vanessa suggested that Smyth should be our bible. what I am doing is to check against Smyth (and sometimes Smith too) the issues with Greek questions status.  Tables have to match Smyth but not necessary Morpheus. If they do not match Morpheus, then it's a Morpheus-data-questionable issue.
table 208 Present Optative (Smyth)
S. 
1. τι-θεί-μην
2. τι-θεῖ-ο 
3. τι-θεῖ-το
D. 2. τι-θεῖ-σθον
3. τι-θεί-σθην 
P. 1. τι-θεί-μεθα
2. τι-θεῖ-σθε 
3. τι-θεῖ-ντο 
or
S. 1. τι-θεί-μην
2. τι-θεῖ-ο
3. τι-θοῖ-το (746 c)
D. 2. τι-θοῖ-σθον
3. τι-θοί-σθην
P. 1. τι-θοί-μεθα
2. τι-θοῖ-σθε
3. τι-θοῖ-ντο

**Comment by monzug** [comment]
in the Smyth book, the above table is 416

**Comment by monzug** [comment]
also, I confirm that the 1st, 2nd and 3rd person singular indicative aorist active are valid forms in Smyth. there is also a note 755 to explain the form in κ
ἔθηκας
ἔθηκα
ἔθηκε  
their stemtype is aor1, it should be emi_aor to match with verbpdgm30 table
